src/distilabel/llm/together.py

```diff
@@ -12,9 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pydantic import PrivateAttr
+import os
+from typing import Optional
+
+from pydantic import Field, PrivateAttr, SecretStr
 
 from distilabel.llm.openai import OpenAILLM
+from distilabel.mixins.runtime_parameters import RuntimeParameter
+
+_TOGETHER_API_KEY_ENV_VAR_NAME = "TOGETHER_API_KEY"
```

Comment:
Why is this defined as global scope while the `TOGETHER_BASE_URL` is not? Shouldn't we align that to either both defined within the class or with global scope instead?